lib/sync/discovery.js

```diff
@@ -16,21 +21,16 @@ module.exports.discovery = (app, queues) => {
       return
     }
 
-    // Store an empty object per repository to indicate that the sync for that repo is pending
+    // Store the repository object to prevent doing an additional query in each job
+    // Also, with an object per repository we can calculate which repos are synched or not
     const repos = repositories.reduce((obj, repo) => {
-      obj[repo.id] = {}
+      obj[repo.node_id] = { repository: getRepositorySummary(repo) }
```

Comment:
Since we're storing the database ID as part of the summary, and because we can use the global `node_id` to look up a repository in GitHub, I changed this.

Reply:
Hmm the problem is that old jobs (the ones that are migrated) need that id to get the repository summary https://github.com/integrations/jira/pull/131/files#diff-9311ea9c09f3ca1ea3c71393232b21feR97

Reply:
Good point. I'll change it back. We can always calculate the `node_id` if we need it anyway.